paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheetViewModel.kt

```diff
@@ -658,8 +680,10 @@ internal class PaymentSheetViewModel @Inject internal constructor(
         }
     }
 
-    private fun setContentVisible(visible: Boolean) {
-        _contentVisible.value = visible
+    private suspend inline fun updateIsAwaitingEagerLaunchResult(crossinline update: (Boolean?) -> Boolean) {
```

Comment:
Should we rename to isAwaiting**Link**EagerLaunchResult? and I'd also add some docs to this function for future reference.

Reply:
I based the naming on the [existing code](https://github.com/stripe/stripe-android/pull/10888/files#diff-f311031c234ce3fc5f9425a479114c55440d6aede03a085e857e0ff3da40320dL275). If we expect future eagerly-launched payment methods to have the same behavior (which I sorta think we do?) then we should keep the more generic naming. I'll defer to you though.